test/hotspot/jtreg/runtime/linkResolver/NoSuchFieldPrimitive.jasm

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+public class NoSuchFieldPrimitive {
+    static int x = 123;
+    static {
+       System.out.println("x = " + x);
+    }
+}
+*/
+
+super public class NoSuchFieldPrimitive
+	version 65:0
+{
+  // REMOVED static Field x:I;
+
+  public Method "<init>":"()V"
+	stack 1 locals 1
+  {
+		aload_0;
+		invokespecial	Method java/lang/Object."<init>":"()V";
+		return;
+  }
+  static Method "<clinit>":"()V"
+	stack 2 locals 0
+  {
+		bipush	123;
+		putstatic	Field x:"I";
+		getstatic	Field java/lang/System.out:"Ljava/io/PrintStream;";
+		getstatic	Field x:"I";
+		invokedynamic	InvokeDynamic REF_invokeStatic:Method java/lang/invoke/StringConcatFactory.makeConcatWithConstants:"(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/String;[Ljava/lang/Object;)Ljava/lang/invoke/CallSite;":makeConcatWithConstants:"(I)Ljava/lang/String;" {
+			String "x = "
+		};
+		invokevirtual	Method java/io/PrintStream.println:"(Ljava/lang/String;)V";
+		return;
+  }
+
+  public static final InnerClass Lookup=class java/lang/invoke/MethodHandles$Lookup of class java/lang/invoke/MethodHandles;
+
+} // end Class NoSuchFieldPrimitive
```

Comment:
These jasm test cases are too complicated and contain a lot of details unrelated to what you're testing. They should be simplified as:

```
/*
public class NoSuchFieldPrimitive {
    // static int x;
    public NoSuchFieldPrimitive() {
       x = 123;
    }
}
*/

super public class NoSuchFieldPrimitive
	version 65:0
{
  // REMOVED static Field x:I;

  public Method "<init>":"()V"
	stack 2 locals 2
  {
		aload_0;
		invokespecial	Method java/lang/Object."<init>":"()V";
		bipush	123;
		putstatic	Field x:"I";
		return;
  }
```